frontend/src/components/GTerminal.vue

```diff
@@ -0,0 +1,914 @@
+        Close
+      </v-btn>
+    </v-snackbar>
+    <v-snackbar
+      v-model="errorSnackbarBottom"
+      :timeout="0"
+      :absolute="true"
+      :auto-height="true"
+      :bottom="true"
+      color="red"
+    >
+      {{ snackbarText }}
+      <v-btn flat @click="hideSnackbar()">
+        Close
+      </v-btn>
+    </v-snackbar>
+    <draggable-component :uuid="uuid">
+      <template v-slot:handle>
+        <v-system-bar dark class="systemBarTop" :class="backgroundClass" @click.native="focus">
+          <v-btn :disabled="!isTerminalSessionCreated" icon small class="text-none grey--text text--lighten-1 ml-0 systemBarButton g-ignore-drag" @click="deleteTerminal">
+            <v-icon>mdi-close</v-icon>
+          </v-btn>
+          <v-spacer></v-spacer>
+          <v-icon class="pr-2">mdi-console</v-icon>
+          <span>{{terminalTitle}}</span>
+          <v-spacer></v-spacer>
+          <v-tooltip v-if="terminalSession.imageHelpText" top class="g-ignore-drag">
+            <!-- g-popper boundariesSelector: The id must not start with a digit. QuerySelector method uses CSS3 selectors for querying the DOM and CSS3 doesn't support ID selectors that start with a digit -->
+            <g-popper
+              slot="activator"
+              :title="`${imageShortText} Help`"
+              toolbarColor="cyan darken-2"
+              :popperKey="`popper_${uuid}`"
+              placement="bottom"
+              :boundariesSelector="`#boundary_${uuid}`"
+            >
+              <span v-html="compiledImageHelpText"></span>
+              <v-btn slot="popperRef" v-if="terminalSession.imageHelpText" icon small class="text-none grey--text text--lighten-1 ml-0 systemBarButton g-ignore-drag">
+                <v-icon>mdi-help-circle-outline</v-icon>
+              </v-btn>
+            </g-popper>
+            Help
+          </v-tooltip>
+          <v-menu
+            bottom
+            offset-y
+            dark
+            min-width="400px"
+          >
+            <template v-slot:activator="{ on }">
+              <v-btn slot="activator" icon small class="text-none grey--text text--lighten-1 ml-0 systemBarButton g-ignore-drag" v-on="on">
+                <v-icon>mdi-menu</v-icon>
+              </v-btn>
+            </template>
+            <v-card tile>
+              <v-card-actions>
+                <v-btn small block slot="activator" flat class="action-button" @click="split('horizontal')">
+                  <icon-base width="16" height="16" viewBox="0 -2 20 20" class="mr-2">
+                    <split-vertically></split-vertically>
+                  </icon-base>
+                  <span>Split Pane Vertically</span>
+                  <v-spacer></v-spacer>
+                  <span>(ctrl + shift + v)</span>
+                </v-btn>
+              </v-card-actions>
+              <v-card-actions>
+                <v-btn small block slot="activator" flat class="action-button" @click="split('vertical')">
```

Comment:
slot versus v-slot

Reply:
we will adress this with https://github.com/gardener/dashboard/issues/630